lib/services/presets.go

```diff
@@ -825,6 +825,34 @@ func NewPresetTerraformProviderRole() types.Role {
 	return role
 }
 
+// NewPresetMCPUserRole returns a new pre-defined role for accessing MCP
+// servers.
+func NewPresetMCPUserRole() types.Role {
+	role := &types.RoleV6{
+		Kind:    types.KindRole,
+		Version: types.V8,
+		Metadata: types.Metadata{
+			Name:        teleport.PresetMCPUserRoleName,
+			Namespace:   apidefaults.Namespace,
+			Description: "Access to MCP servers",
+			Labels: map[string]string{
+				types.TeleportInternalResourceType: types.PresetResource,
+			},
+		},
+		Spec: types.RoleSpecV6{
+			Allow: types.RoleConditions{
+				AppLabels: map[string]apiutils.Strings{
+					types.Wildcard: []string{types.Wildcard},
```

Comment:
> Improve day zero experience so the end user doesn't have to think about RBAC too much.

Why we don't suggest to use the preset `access` role instead ? 


It there a way to scope down `types.Wildcard: []string{types.Wildcard},`  to  MCP apps  based on the appLabel ? 

I’m raising this because using AppLabels: types.Wildcard can lead to unintended side effects. For example, the SAML IdP app ends up effectively hidden behind the app_label=RBAC, due to the Okta app’s upstream Assignment provisioning.

Reply:
>  i can implement this is to have inject some internal labels for all MCP apps, to be used for app_labels

Agree - If we can take this route, it would give clients a much safer and more predictable expectation what `mpc-user` can do within teleport  cluster.

Reply:
Yes, that looks perfect. I’m just wondering whether we want to apply this to all apps with subKind != "", or limit this logic to the MCP app for now.